api/app/controllers/transcript.controller.ts

```diff
@@ -153,23 +188,28 @@ export async function archive(req: Request, res: Response) {
     return;
   }
 
-  Transcript.update(
-    { archivedAt: new Date(), archivedBy: userId },
-    {
-      where: { id: Number(id) },
-    }
-  )
-    .then((num) => {
+  try {
+    const responseData = await Transcript.update(
+      { archivedAt: new Date(), archivedBy: userId },
+      {
+        where: { id: Number(id) },
+      }
+    );
+    if (responseData[0] === 0) {
```

Comment:
We are checking for 0 here and 1 in line 142. Do we need to check for these exact `numbers`, if anything goes wrong the catch block should catch it.

Reply:
Yes, the numbers are intentional. We get a 0 or 1 response from the db. 0 means the operation was unsuccessful and 1 means it was successful